Updates to tut2_EOFM_Evaluation

lillythomas opened the pull request:
Made functional for opening in colab.

lillythomas committed:
add pip install, fix comment, add more markdown instructions

lillythomas committed:
add netrc file creation step

lillythomas committed:
add open in collab button

weiji14 commented on tut2_EOFM_Evaluation.ipynb:
```diff
@@ -107,9 +136,28 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!echo \"machine urs.earthdata.nasa.gov login your_username password your_password\" > ~/.netrc\n",
+    "!chmod 0600 ~/.netrc"
```
The [earthaccess.login(persist=True)](https://earthaccess.readthedocs.io/en/latest/user_guide/authenticate/#login-using-a-netrc) line will create a netrc file, so no need to explicitly add this.

weiji14 reviewed (approved):
Thanks @lillythomas, verified that it runs on Google Colab with a T4 GPU (though there was one instance when I hit into some `CPLE_OpenFailedError: '/vsicurl/https:// ... .tif' not recognized as a supported file format` that was fixed with a restart+retry). Just one suggestion to remove the netrc instructions since `earthaccess` should handle it.

weiji14 committed:
Remove manual netrc creation as earthaccess.login(persist=True) does it

Xref https://earthaccess.readthedocs.io/en/latest/user_guide/authenticate/#login-using-a-netrc

weiji14 committed:
Update placeholders to point to tut2_EOFM_Evaluation.ipynb

lillythomas commented:
@weiji14 I wasn't able to get the `.netrc` file from  `earthaccess.login(persist=True)` alone on colab actually, which is why I added the manual command. In the absence of that, I would get the same permissions error you hit `CPLE_OpenFailedError: /vsicurl/https:// ... .tif not recognized as a supported file format` but I guess if you were able to avoid that after restarting, maybe it means the that the machine needs to refresh after to see the updates to the file system from  `earthaccess.login(persist=True)`.

weiji14 commented:
> @weiji14 I wasn't able to get the `.netrc` file from `earthaccess.login(persist=True)` alone on colab actually, which is why I added the manual command. In the absence of that, I would get the same permissions error you hit `CPLE_OpenFailedError: /vsicurl/https:// ... .tif not recognized as a supported file format` but I guess if you were able to avoid that after restarting, maybe it means the that the machine needs to refresh after to see the updates to the file system from `earthaccess.login(persist=True)`.

Hmm, did you check in `~/.netrc` instead of the current working directory? I did try deleting the `~/.netrc` to be sure... Could you try without the `persist=True` flag, I'm actually wondering if we need to netrc if earthacess handles the auth already.
